src/yaml2json.cc

```diff
@@ -0,0 +1,21 @@
+#include "yaml2json.h"
+#include <glib.h>
+#include <logging/logging.h>
+
+Yaml2Json::Yaml2Json(const std::string& yaml) {
+  g_autofree gchar* stdout_buff = nullptr;
+  g_autofree gchar* stderr_buff = nullptr;
+  g_autofree GError* error = nullptr;
+  gint status;
+
+  // clang-tidy insane line wrapping requirements follows
+  if (g_spawn_command_line_sync(("fy-tool --mode json " + yaml).c_str(), &stdout_buff, &stderr_buff, &status, &error) !=
```

Comment:
libaktualizr has a helper to make this easier: https://github.com/foundriesio/aktualizr/blob/c31f3956277fb6c9dcdc4a7aa7a61d6f3dc07964/src/libaktualizr/utilities/utils.h#L42

Reply:
yes; just changed.
also we should probably be giving full paths to the binaries (so what gets executed is not affected by chages in $PATH)